sdk/storage/blobs/src/az_storage_blobs_blob_client.c

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_http_header.h>
+#include <az_json_builder.h>
+#include <az_storage_blobs.h>
+
+#include <_az_cfg.h>
+
+/**
+ * @brief Example URL syntax, not all URLs will be in this format
+ * Basic url is expected as : [https://]{account_id}[.blob.core.windows.net]/{container}/{blob}{optional_query}
+ * URL token                       max Len            Total
+ * [https://]                       = 8                 8
+ * {account_id}                     = 24               32  // 3-24 lowercase only
+ * [.blob.core.windows.net]         = 22               54  //TODO: Support soveriegn clouds
+ * /{container}                     = 64              118  // 1 + 3-63, lowercase letters/numbers/- (consecutive dash is invalid)
+ * /{blob}                          = 1025           1143  // 1 + 1024
+ * {optional_query}                 = 70          ** 1213 **
+ */
+enum { MAX_URL_SIZE = 1280 };       // Padded to 1280
+enum { MAX_BODY_SIZE = 1024 * 10 }; // 10KB buffer  /*TODO, Adjust this to reasonable size for non-stream data.*/
+
+AZ_NODISCARD AZ_INLINE az_span az_storage_blobs_client_constant_blob_type() { return AZ_STR("x-ms-blob-type");}
+
+az_storage_blobs_blob_client_options const AZ_STORAGE_BLOBS_BLOB_CLIENT_DEFAULT_OPTIONS
+    = { .service_version = AZ_CONST_STR("2015-02-21"),  //TODO - Standardize handling of Service versions
+        .retry = {
+            .max_retry = 3,
+            .delay_in_ms = 30,
```

Comment:
>delay_in_ms = 30 [](start = 13, length = 16)

I noticed that this value is copied from `az_keyvault_client`, however this is not so ideal a value for azure storage services. Normally the delay would be longer than a second when retrying.

Reply:
The value can certainly be modified.  As a representative from storage what would you suggest using for retry values?